module-cheatsheets/intro-to-R-tidyverse-cheatsheet.md

```diff
@@ -9,74 +9,97 @@
-### Docker 
-Read documentation on the Docker container [**here**](https://www.docker.com/resources/what-container). 
-
-Piece of code                                   | What it's called                                | What it does                           |
-|-------------------------------------------------|------------------------|--------------------------------------------------------------------------------------|
-| [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) | Docker Pull      | Pulls an image from a docker container |
-| [`docker run`](https://docs.docker.com/v17.12/engine/reference/run/)  | Docker Run             | Runs processes in a docker container   |
-
-
-### `ggplot2`
-Read the `ggplot2` package documentation [**here**](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0). <br>
+#### `ggplot2`
+Read the `ggplot2` package documentation [**here**](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0). <br>
 A vignette on the usage of the `ggplot2` package can be found [**here**](https://cran.r-project.org/web/packages/ggplot2/vignettes/ggplot2-specs.html).
 
 | Library/Package                      | Piece of code                                   | What it's called       | What it does                                                                                                                                                                                      |
 |--------------------------------------|-----------------------------------------------------|--------------------------|--------------------------------------------------------------------------------------------------------------------------------------|
-| `ggplot2`         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
-| `ggplot2`         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.1.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when combined with ggplot()            |
+| `ggplot2`         | [`ggplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/ggplot)                   | GG Plot               | Begins a plot that is finished by adding layers.        |
+| `ggplot2`         | [`aes()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/aes)                   | Aesthetic Mappings              | Designates how variables in the data object are mapped to the visual properties of the ggplot.        |             
+| `ggplot2`         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when combined with ggplot().            |
```

Comment:
```suggestion
| `ggplot2`         | [`geom_boxplot()`](https://www.rdocumentation.org/packages/ggplot2/versions/3.3.0/topics/geom_boxplot)       | Boxplot               | Creates a boxplot when added as a layer to a `ggplot()` object.            |
```

Reply:
Can you use these kinds of words through the ggplot2 layer functions. Wanna be consistent in referring to them as being added on as more layers. `combined` I think is unclear.